src/composition.cc

```diff
@@ -116,6 +116,18 @@ Composition::Composition(int prev_decay, ChainPtr decay_line)
   next_id_++;
 }
 
+std::string Composition::ToString(CompMap v) {
+  std::string comp = "{";
```

Comment:
Slight readability improvements of the final product, without bloating it too much.  Adding a space and using a different delimiter than each of the nuclide entries
```suggestion
  std::string comp = "[ ";
```